tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -7188,6 +7188,22 @@ TEST_F(SQLiteFixture, OutputReportTabular_WriteLoadComponentSummaryTables_AirLoo
     // same Design Days peak and timestep peak as the zone it serves. This is the critical part of the test
     state->dataSize->FinalSysSizing(1).loadSizingType = DataSizing::LoadSizing::Total;
 
+    // For #9772
+    state->dataSize->NumSysSizInput = 1;
```

Comment:
Now the loop where you gather the mixed temps will exectute and capture the correct data.